attributes/default.rb

```diff
@@ -58,8 +58,9 @@
 #
 # Filebeat
 
-default['filebeat']['version']                  = "7.2.0"
-default['filebeat']['url']                      = "#{node['download_url']}/filebeat-oss-#{node['filebeat']['version']}-linux-x86_64.tar.gz"
+default['filebeat']['version']                  = "7.12.1"
```

Comment:
why not `7.13.2` same as logstash?
https://www.elastic.co/downloads/past-releases/filebeat-oss-7-13-2